webapp/src/App.tsx

```diff
@@ -8,6 +8,23 @@ import ReactDefaultApp from './ReactDefaultApp';
  * This is the main App component. If this starts getting too big, remember to refactor and nest things!
  * @returns
  */
+
+/* TODO * Will delete these Sound Effects later since it was just for fun! */
```

Comment:
Since you put this as a `TODO` did you plan to do that prior to the PR or is that a not fore Future-Signe?

Reply:
Yes, I should have added that it was for future Signe! I plan to delete the sound effect functions() after this PR is approved since we won't need them for the project.🙂